index.bs

```diff
@@ -600,25 +600,26 @@ To <dfn>run the update intersection observations steps</dfn> for a
 			algorithm on |target|.
 		5. Let |targetArea| be |targetRect|'s area.
 		6. Let |intersectionArea| be |intersectionRect|'s area.
-		7. Let |intersectionRatio| be |intersectionArea| divided by |targetArea| if |targetArea| is non-zero,
+		7. Let |intersectionRatio| be |intersectionArea| divided by |targetArea|
+			if |targetArea| is non-zero,
 			and <code>0</code> otherwise.
-		8. If |intersectionRatio| is equal to <code>0</code>,
-			let |threshold| be <code>0</code>.
-			If |intersectionRatio| is equal to <code>1</code>,
-			let |threshold| be the length of |observer|.{{thresholds}}.
-			Otherwise, let |threshold| be the index of the first entry in |observer|.{{thresholds}}
-			whose value is greater than |intersectionRatio|.
-		9. Let |intersectionObserverRegistration| be the {{IntersectionObserverRegistration}} record 
+		8. Let |doesIntersect| be true if |targetRect| and |rootBounds| intersect or are edge-adjacent,
+			even if the intersection has zero area (because |rootBounds| or |targetRect| have
+			zero area); otherwise, let |doesIntersect| be false.
+		9. If |doesIntersect| is false, let |thresholdIndex| be <code>0</code>.<br>
+			Otherwise, if |targetRect| has zero area, let |thresholdIndex| be <code>1</code>.<br>
```

Comment:
Instead of 1, should this be "the length of |observer|.{{thresholds}}"? I know our current code does 1, but we had talked of exposing the threshold index on the IntersectionObserverEntry. I don't know if we should do that, but if we did, then we would probably want to change this.